picard/ui/ui_options_tags_compatibility_aac.py

```diff
@@ -28,15 +34,16 @@ def setupUi(self, TagsCompatibilityOptionsPage):
         self.remove_ape_from_aac.setObjectName("remove_ape_from_aac")
         self.verticalLayout.addWidget(self.remove_ape_from_aac)
         self.vboxlayout.addWidget(self.aac_tags)
-        spacerItem = QtWidgets.QSpacerItem(274, 41, QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Expanding)
-        self.vboxlayout.addItem(spacerItem)
+        spacerItem1 = QtWidgets.QSpacerItem(274, 41, QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Expanding)
+        self.vboxlayout.addItem(spacerItem1)
 
         self.retranslateUi(TagsCompatibilityOptionsPage)
         QtCore.QMetaObject.connectSlotsByName(TagsCompatibilityOptionsPage)
 
     def retranslateUi(self, TagsCompatibilityOptionsPage):
         _translate = QtCore.QCoreApplication.translate
         self.aac_tags.setTitle(_("AAC files"))
+        self.info_label.setText(_("Picard can save APEv2 tags to pure AAC files, which by default do not support tagging. APEv2 tags in AAC are supported by some players (e.g. foobar2000 or MusicBee), but players not supporting AAC files with APEv2 tags can have issues loading and playing those files. Most often they display a wrong duration, causing issues on track change. To deal with this you can choose whether to save tags to those files."))
```

Comment:
I would not cite players _(e.g. foobar2000 or MusicBee)_ because it isn't future-proof.

Reply:
Fair point. I removed those references.